test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -830,7 +846,6 @@ public void SaveMulticlassLrPredictorAsSummary()
             _step++;
             OutputPath summaryModel = CreateOutputPath("summary.txt");
             TestInCore("savepredictor", null, trainModel, null, summaryModel.Arg("sum"));
-
             Done();
```

Comment:
Nitpick. Can the blank like be added back?  Otherwise a high level view does not reflect true view of the changed areas.